atomic_reactor/util.py

```diff
@@ -1291,6 +1292,15 @@ def read_yaml(yaml_data, schema):
     return data
 
 
+def allow_repo_dir_in_dockerignore(build_path):
+    docker_ignore = os.path.join(str(build_path), DOCKERIGNORE)
+
+    if os.path.isfile(docker_ignore):
+        with open(docker_ignore, "a") as f:
+            f.write("!%s\n" % RELATIVE_REPOS_PATH)
```

Comment:
What if .dockerignore does not already end in a newline character? I think we may want to add one ourselves before the `!`.

Reply:
sure